README.md

```diff
@@ -64,6 +64,23 @@ If you don't want to change configuration, but just want to update an existing p
 
 Turn prompts off with: `yo videojs-plugin --hurry`
 
+#### Limiting Generated Files
+
+In some cases - especially when updating previously generated projects - you may want to only update certain "meta" files without needing to deal with prompts for source files and test files you definitely do not want to overwrite. The `--limit-to` and `--limit-to-meta` option helps with this.
+
+`--limit-to` accepts keys which will limit the generated files to only those specified. The possible keys are:
+
+- `dotfiles`: Updates _only_ those files starting with a `.` and `bower.json`.
```

Comment:
It does not include `package.json`. But it does include `bower.json` if you have bower enabled. Perhaps `bower.json` should be included in the `pkg` option instead since it's not a dotfile...